hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/states/endpoint/VersionEndpointTask.java

```diff
@@ -75,6 +76,10 @@ public EndpointStateMachine.EndPointStates call() throws Exception {
           // If end point is passive, datanode does not need to check volumes.
           String scmId = response.getValue(OzoneConsts.SCM_ID);
           String clusterId = response.getValue(OzoneConsts.CLUSTER_ID);
+          DatanodeLayoutStorage layoutStorage
+              = new DatanodeLayoutStorage(configuration, clusterId);
+          layoutStorage.setClusterId(clusterId);
+          layoutStorage.forceInitialize();
```

Comment:
@errose28, Thanks for the review. I tried using `DatanodeLayoutStorage#persistCurrentState` but it throws `Metadata dir doesn't exist`.

Reply:
Is it throw from the below testDnLayoutVersionFile test? It's because the DatanodeLayoutStorage is not initialized in the test.  If it's initialized, persistCurrentState will work.

Reply:
Used `DatanodeLayoutStorage#persistCurrentState` and fixed the test cases.